application/src/main/resources/application-auth-dev.properties

```diff
@@ -1,4 +1,5 @@
 # Authentication
+spring.security.oauth2.resourceserver.jwt.issuer-uri=http://localhost:8080/realms/aicockpit
```

Comment:
This must be `http://localhost:8080/auth/realms/ai-cockpit` to make it work with the `auth-docker-compose.yml`